bin/desimodel_sync_kpno.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+
+# This script is intended to be run from a cronjob (see 
+# desimodel/etc/desimodel_sync_kpno_cron.sh).  You should not run
+# this script interactively.  It expects to be run inside an
+# already-configured software environment.
```

Comment:
If this is only intended to be run via the cronjob, does it need to be a separate script that gets installed into $PATH along with other desimodel/bin scripts, or could it just be the last few lines of the `etc/desimodel_sync_kpno_cron.sh` script itself?

Either way, let's put a safety check in that it will only run as datasystems@desi-7.kpno.noao.edu (perhaps with a force option or magical environment variable override if needed) to prevent

Reply:
Ok, that sounds reasonable to squash the two together, since the second script is just finding the latest calibration file.